packages/lesswrong/components/editor/EditUrl.jsx

```diff
@@ -46,7 +50,7 @@ class EditUrl extends Component {
           </span>
           <span className={classNames(classes.input, {[classes.hideInput]: !active})}>
             <Components.MuiTextField
-              {...this.props}
+              {...propsCopy}
```

Comment:
That's not how you are supposed to do that. I think this is the correct pattern: 

https://stackoverflow.com/questions/35152522/react-transferring-props-except-one

I.e.: 

```
render() {
  var {one, ...other} = this.props;
  return (
    <Cpnt {...other}>
      <Subcpnt one={one} />
    </Cpnt>
  );
}
```